src/OrchardCore.Modules/OrchardCore.Contents/Workflows/Activities/CreateContentTask.cs

```diff
@@ -63,7 +63,9 @@ public async override Task<ActivityExecutionResult> ExecuteAsync(WorkflowExecuti
                 var contentProperties = await _expressionEvaluator.EvaluateAsync(ContentProperties, workflowContext);
                 var propertyObject = JObject.Parse(contentProperties);
 
-                ((JObject)contentItem.Content).Merge(propertyObject);
+                var contentItemEvaluated = propertyObject.ToObject<ContentItem>();
+                contentItem.DisplayText = contentItemEvaluated.DisplayText;
+                contentItem.Apply(contentItemEvaluated);
```

Comment:
If the DisplayText isn't coming through here, then maybe my advise was wrong.
Perhaps it needs to be `var contentItemFromJson = JsonConvert.DeserializeObject<ContentItem>(contentProperties);`

This would use the `[ContentItemConverter]` attribute to convert the json, much as if it came out of the database

Reply:
No the issue is that contentItem.Apply(contentItemEvaluated) only affects contentItem.Data items. And contentItem.DisplayText is not part of it. So we need to manually affect it.

Reply:
```
        /// <summary>
        /// Updates the whole content.
        /// </summary>
        /// <typeparam name="element">The content element instance to update.</typeparam>
        /// <returns>The current <see cref="ContentItem"/> instance.</returns>
        public static ContentElement Apply(this ContentElement contentElement, ContentElement element)
        {
            if (contentElement.Data != null)
            {
                contentElement.Data.Merge(JObject.FromObject(element.Data), JsonMergeSettings);
            }
            else
            {
                contentElement.Data = JObject.FromObject(element.Data, ContentBuilderSettings.IgnoreDefaultValuesSerializer);
            }

            contentElement.Elements.Clear();
            return contentElement;
        }
```

Reply:
Yup. Makes sense. My only concern on setting DisplayText manually was about the other content item properties. But I'm not sure you need to be able to set them through a workflow, as I am not super familiar with it. LGTM though

Reply:
I  think the proper fix though would be to move displayText in contentItem.Content ... not sure.